packages/drivers/routerlicious-driver/src/routerliciousResolvedUrl.ts

```diff
@@ -0,0 +1,30 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { IResolvedUrl } from "@fluidframework/driver-definitions";
+
+/**
+ * Routerlicious extends the resolved url with additional properties to control Routerlicious-specific behaviors.
+ * @alpha
+ */
+export interface IRouterliciousResolvedUrl extends IResolvedUrl {
+	/**
+	 * A flag to facilitate type narrowing from IResolvedUrl to IRouterliciousResolvedUrl.
+	 */
+	routerliciousResolvedUrl: true;
+	/**
+	 * Controls whether a newly created container will be ephemeral. Only affects createContainer requests.
```

Comment:
Can we be more explicit about what "ephemeral" means in this context? If there is other documentation we can link to, that would probably be sufficient. We should also use a `@defaultValue` block to define the meaning of `undefined` here.

Reply:
I would also recommend using an `{@link}` tag to point to the definition of `createContainer` here if possible.

Reply:
Done in latest!